---
layout: page
title: Syllabus
description: Course Syllabus
nav_order: 2
---

# Syllabus
{:.no_toc}

## Table of contents
{: .no_toc .text-delta }

1. TOC
{:toc}

---

Anyone is welcome to attend and participate in our Journal Club. Students seeking to receive credit for Journal Club have specific responsibilities described below.

## Scope of the Journal Club
The Computational Molecular Biology Journal Club is a forum for discussing important and recent research results in computational and systems biology. Biological areas of interest to this seminar are wide-ranging, including (but not limited to) genomics and other -omics, metabolic and regulatory models, discovery of genotype-phenotype associations, phylogenetics and evolution, metagenomics, and synthetic biology. The unifying theme is that the works being studied should make a substantial computational or formal contribution. This contribution may be analytic -- e.g. data mining, data analysis, and hypothesis generation -- or synthetic -- e.g. modeling, prediction, and design of new experiments. Papers that rely heavily on existing, nontrivial computational methods to produce important biological results are also welcome but should be presented together with an explanation of these underlying methods.

Here is an extremely non-exhaustive list of conferences and journals that feature research relevant to the Journal Club:

- Science;
- Nature and its sub-journals, such as Nature Biotechnology;
- Bioinformatics (Oxford University Press), PLOS Bioinformatics, and BMC Bioinformatics;
- Journal of Computational Biology;
- IEEE Transactions on Computational Biology and Bioinformatics;
- Genome Research; Genome Biology
- Nucleic Acids Research;
- Proceedings of the National Academy of Sciences (PNAS);
- The Conference on Research in Computational Molecular Biology (RECOMB) and its many satellite conferences;
- The International Conference on Intelligent Systems for Molecular Biology (ISMB), and its sister forum, the European Conferences\ on Computational Biology (ECCB);
- The Pacific Symposium on Biocomputing (PSB).

## Student Responsibilities
The following information applies only to students who are taking Journal Club for seminar credit, whether graded (for DBBS students) or ungraded (for CSE students).

To receive credit for Journal Club, you must do three things:

- Present a paper in at least one meeting of the Journal Club;
- Attend the other meetings and remain engaged in discussion;
- Submit written responses to each week's paper, prior to the meeting at which it is presented.

These three factors are weighted roughly equally in assigning credit and grades.

## Paper Selection
The range of topics for each semester of Journal Club is negotiated at an initial organizational meeting. Individual papers and presentation dates are selected by each student, with suggestions and final approval from the course masters.

Each student must select a paper at least five days before the meeting in which it is presented, to give others time to read the paper and submit responses.

## Written Responses
To ensure that everyone is prepared to participate in each meeting at the highest level, we ask that students submit a written response to each scheduled paper in the week before it is presented. (If you are presenting that week, you don't need to submit a written response.) Written responses should be posted to the Journal Club's Canvas discussion forum. To reach the forum, go to [mycanvas.wustl.edu](mycanvas.wustl.edu), log in, and select the course "BIOL.5496.01". (You should select this course even if you are enrolled in CSE7800.) Go to the discussions page, which will show the "Paper Responses" forum where you will post your responses.

You will not be able to see others' responses until you post your own; after that, you can read and reply to any other response, and the other students will be able to read yours.

Written responses should be around a page in length. They should include three components: a brief summary of the paper, an assessment of its key strengths and weaknesses, and a reflection on what steps to take next. Your assessment should focus on the following areas:

- **Significance:** what questions do the authors pose, what claims do they make, and what do they want you to believe? Why is the world a better place for this work having been done? What is the work's potential impact on the field?
- **Innovation:** how does this work increase the totality of the discipline's knowledge and/or skills? How does it relate to previous work in the same area?
- **Approach:** what are the strengths and limitations of the methods invented and/or employed? Are they correct, efficient, and appropriate to the questions being asked? How might they bias the results? Are there issues of statistical significance and power, and if so, are they addressed appropriately?
- **Validation:** does the paper advance evidence that leads you to believe its claims? For methods papers, is there validation on real (vs. simulated) data, and are there appropriate comparisons to other methods? Identify the key figures in the paper and what each one communicates.

To the extent possible, please try to identify both strengths and weaknesses in every paper. Papers that pass peer review in respected forums generally have some value, whatever their limitations. The goal is not to "shoot down" a paper but rather to assess what can usefully be taken away from it.

Finally, after assessing the paper, you should consider and respond to the following question. If you were a student joining the authors' research group and had to take over the project described in the paper, what would your next step be to continue the research? This could be additional validation, extension of a method, followup experiments, statistical tests, or whatever seems most useful.

The key in answering this question is to propose a specific, well-justified course of action. For example, don't just say, "I would perform further testing on my HMM to check its specificity;" instead, say "I would test my HMM on the SCOP database and draw ROC curves to assess its performance on beta-barrel proteins in particular because beta barrel proteins have unusual sequence composition. Alternative training sets may be required for optimal performance on beta barrel proteins."

